jsettlers.logic/src/main/java/jsettlers/logic/player/PlayerSetting.java

```diff
@@ -1,46 +1,75 @@
 package jsettlers.logic.player;
 
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
 import java.util.Arrays;
+import java.util.Random;
 
 import jsettlers.common.CommonConstants;
 import jsettlers.common.ai.EPlayerType;
 import jsettlers.common.player.ECivilisation;
 
 /**
  * @author codingberlin
+ * @author Andreas Eberle
  */
 public class PlayerSetting {
 
-	private final EPlayerType playerType;
+	private static final short INITIAL_VERSION = 1;
+
+	private final boolean available;
+	private final Byte teamId;
```

Comment:
Then it should be fine to use this (although we could e.g. use negative numbers instead).